src/components/PetCard.js

```diff
@@ -30,7 +31,7 @@ export default function PetCard(props) {
 
   return (
     <div className='border rounded m-1 p-2' onClick={onClickCard}>
-      <img src={pet.Picture} />
+      <img className="img-fluid rounded" src={HOSTNAME + pet.Picture} width="100%"/>
```

Comment:
For some reason, `HOSTNAME` was deleted from the previous PR. Adding it back here.